### DISCUSSION TITLE
Creating comparison charts

### DISCUSSION THREAD

**Discussion by saimarshadsaim31** [FormidableLabs/victory-native-xl#360] [Q&A] [answered]
Is there a way that we can create comparison charts that show two lines representing a comparison. I am talking about something like this 
![image](https://github.com/user-attachments/assets/cfb4d0f8-50f7-448f-902e-05e71840a231)
Also is there a way i can set opposite y labels for each line as shown in the image

**Comment by zibs** [comment, marked as answer]
This should be possible once #352 is merged and release (very soon!)
